optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -258,6 +258,17 @@ def _save_pretrained(self, save_directory: Union[str, Path]):
             self.feature_extractor.save_pretrained(save_directory / "feature_extractor")
 
         self._save_openvino_config(save_directory)
+        self._save_pipeline_config(save_directory)
+
+    def _save_pipeline_config(self, save_directory):
+        model_dir = (
+            self.model_save_dir
+            if not isinstance(self.model_save_dir, TemporaryDirectory)
+            else self.model_save_dir.name
+        )
+        original_config = Path(model_dir) / self.config_name
+        if original_config.exists():
+            shutil.copy(original_config, save_directory)
```

Comment:
makes more sense if this is under `_save_config`, to avoid saving the first and then overwriting it the copied one.
https://github.com/huggingface/optimum/blob/58c3571156466c13ebc7e22d3405b376ab2d222b/optimum/modeling_base.py#L137C1-L140C46

Reply:
ok updated